recipes/ffmpeg/all/conanfile.py

```diff
@@ -108,23 +108,19 @@ def config_options(self):
             del self.options.with_pulse
         if self.settings.os != "Macos":
             del self.options.with_appkit
-            del self.options.with_avfoundation
+        if self.settings.os not in ["Macos", "iOS", "tvOS"]:
```

Comment:
Would `tools.is_apple_os` do the job too?
```suggestion
        if not tools.is_apple_os(self.settings.os):
```

Reply:
No, because watchOS must be excluded here